tests/test.py

```diff
@@ -11,6 +11,20 @@ class Test:
     """
     A class for writing and running tests with a fluent API.
 
+    Attributes:
+        title (str): The title of the test.
+        mode (Literal["soft", "hard"]): The mode of the test. If set to "hard", the test will exit upon the first failure.
+        export_file_name (str): The name of the file where the test results will be exported.
+        result (str): The result of the test, formatted as a Markdown string.
+        test_obj: The object that is currently being tested.
+
+    Methods:
+        expect(obj): Set the object to be tested.
+        to_be(expected, msg: str | None = None): Assert that the test object is equal to the expected value.
+        to_be_of_type(obj_type, msg: str | None = None): Assert that the test object is an instance of the expected type.
+        to_be_approximately(expected, precision, msg: str | None = None): Assert that the test object is approximately equal to the expected value within a given precision.
```

Comment:
"Why not "is_approximately" rather "to_be_approximately". Short declarative names are better. Same comment for the other functions.

Reply:
Will update that.

Reply:
Completed in commit https://github.com/anand-kamble/data-filtering/pull/9/commits/2a83b5701d708ed985be33b130892e05e438acea